README.md

```diff
@@ -286,6 +167,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please use the appropriate license as this link is currently broken :+1: :sparkles: